userspace/falco/configuration.cpp

```diff
@@ -238,12 +250,28 @@ void falco_configuration::load_engine_config(const std::string& config_name)
 
 void falco_configuration::load_yaml(const std::string& config_name)
 {
+	init_logger();
 	load_engine_config(config_name);
-	m_log_level = config.get_scalar<std::string>("log_level", "info");
 
 	std::list<std::string> rules_files;
 
-	config.get_sequence<std::list<std::string>>(rules_files, std::string("rules_file"));
+	// Small glue code to support old deprecated 'rules_file' config key.
+	int num_rules_files_opts = 0;
+	if (config.is_defined("rules_files"))
+	{
+		num_rules_files_opts++;
+		config.get_sequence<std::list<std::string>>(rules_files, std::string("rules_files"));
+	}
+	if (config.is_defined("rules_file"))
+	{
+		num_rules_files_opts++;
+		config.get_sequence<std::list<std::string>>(rules_files, std::string("rules_file"));
+		falco_logger::log(falco_logger::level::WARNING, "Using deprecated config key 'rules_file'. Please use new 'rules_files' config key.");
```

Comment:
Warning without a deadline; we might want to keep the old config key for many years (or just until Falco 1.0.0).

Reply:
I quite liked the version with more info:

```suggestion
		falco_logger::log(falco_logger::level::WARNING, "Using deprecated config key 'rules_file' (singular form). Please use new 'rules_files' config key (plural form).");
```